js_utils/text_validation.js

```diff
@@ -56,6 +56,12 @@ export class TextValidation {
       exception.err = "Not Supported " + exception.err;
       throw exception;
     }
+    if (
+      (text && type === "plate_barcode" && !text.includes("ML")) ||
+      (text && type === "stim_barcode" && !text.includes("MS"))
+    ) {
+      feedback = "barcode type no match";
+    }
```

Comment:
sorry not to be nit-picky, but this should go in `validate_plate_barcode` since it's just for validating barcodes on line 181

Reply:
I left it here because I thought we want to validate both stim and plate barcodes ? Do we only want to validate plate barcode? @tannermpeterson

Reply:
If you look in that function, it checks for both MS and ML headers, so it's just named incorrectly. We still need the check to go inside it